WooCommerce/UITestsFoundation/Screens/TabNavComponent.swift

```diff
@@ -11,16 +11,21 @@ public final class TabNavComponent: ScreenObject {
         $0.tabBars.firstMatch.buttons["tab-bar-orders-item"]
     }
 
-    private let reviewsTabButtonGetter: (XCUIApplication) -> XCUIElement = {
-        $0.tabBars.firstMatch.buttons["tab-bar-reviews-item"]
-    }
-
     private let productsTabButtonGetter: (XCUIApplication) -> XCUIElement = {
         $0.tabBars.firstMatch.buttons["tab-bar-products-item"]
     }
 
+    private let menuTabButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.tabBars.firstMatch.buttons["tab-bar-menu-item"]
+    }
+
+    private let reviewsTabButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.tabBars.firstMatch.buttons["tab-bar-reviews-item"]
+    }
+
```

Comment:
Is this still kept for backwards compatibility?

Reply:
Oh right, I can remove this, there was an attempt to use feature flag but that didn't work.